.github/workflows/build-from-main.yml

```diff
@@ -0,0 +1,310 @@
+name: Check and Build Main
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+  packages: write
+
+jobs:
+  check-build:
+    if: contains(fromJson('["main", "staging"]'), github.ref_name)
+    environment: ${{ matrix.network }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - branch: main
+            network: preview-govtool
+            workdir: ./govtool/backend
+            name: govtool-backend
+            dockerfile: ./govtool/backend/Dockerfile.qovery
```

Comment:
I think we can get rid of base Dockerfiles in favor of this .qovery ones (that would need adjustments to test-stack workflows). What do you think @mesudip?